src/v/redpanda/admin/security.cc

```diff
@@ -580,6 +582,38 @@ admin_server::oidc_revoke_handler(std::unique_ptr<ss::http::request>) {
     co_return ss::json::json_return_type(ss::json::json_void());
 }
 
+ss::future<ss::json::json_return_type> admin_server::list_user_roles_handler(
+  std::unique_ptr<ss::http::request> req, request_auth_result auth_result) {
+    ss::sstring filter = req->query_parameters["filter"];
+
+    auto member = [&auth_result]() -> std::optional<security::role_member> {
+        if (!auth_result.is_authenticated()) {
+            return std::nullopt;
+        }
+        return security::role_member{
+          security::role_member_type::user, auth_result.get_username()};
+    }();
+
+    if (!member.has_value()) {
+        throw ss::httpd::base_exception{
```

Comment:
Do we need a debug line here or is that done further up when the exception is caught?

Reply:
Fair point. Yeah, it does look like it'll get logged further up the chain:

https://github.com/redpanda-data/redpanda/blob/96036cad8ad7301733b2f39475c2617462a99d77/src/v/redpanda/admin/server.cc#L644-L653

Which I think is sufficient here because there's nothing else to report about the state of the broker.